napari/layers/image/image.py

```diff
@@ -26,8 +27,10 @@
     SliceDataClass = ImageSliceData
 
 
+# It is important to contain at least one abstractmethod to properly exclude this class
```

Comment:
I like this a lot as an exclusion criterion for `NAMES`... but I'm not sure all of this is necessary. I pulled this locally, removed all of the abc imports and stuff from `image.py`, and removed the abstractmethods here from `_ImageBase` and `inspect.isabstract(_ImageBase)` is _still_ true, and NAMES still excludes `_ImageBase` (simply because `_get_state` from the base layer hasn't been implemented).

So can we remove the metaclass and the abstract methods.  Remove the redundant `_get_state` method, from `_ImageBase`, move `Image._raw_to_displayed` back up to `_ImageBase`, and then remove this comment here?

Reply:
I first add metaclass and then move functions to the subclass. 
but `_raw_to_displayed` is used in `_new_empty_slice` so I think that it should be declared as `abstractmethod` because `Image` nad `Layer` implements `_raw_to_displayed` in different way.

Reply:
Image basically doesn't even implement it.  It's just a no-op.  It's fine for a subclass to re-implement something from a base class in a different way.  If we don't put it on `_ImageBase`, then any other class is going to have to make another silly `def _raw_to_displayed(self, raw): return raw`... which is even weirder

Reply:
ok